common/law_groups/00_babylon_law_groups.txt

```diff
@@ -1,7 +1,7 @@
 ﻿lawgroup_diplomatic_stance = {
 	base_enactment_days = 180
```

Comment:
It can also greatly be sped up like maybe 30 days or something. I was planning to do something like that for tweaks.

Reply:
Fixed, ready for second review @HiddeLekanne